kubetest/extract.go

```diff
@@ -68,16 +68,16 @@ func (l *extractStrategies) String() string {
 // Converts --extract=release/stable, etc into an extractStrategy{}
 func (l *extractStrategies) Set(value string) error {
 	var strategies = map[string]extractMode{
-		`^(local)`:                  local,
-		`^gke-?(staging|test)?$`:    gke,
-		`^gci/([\w-]+)$`:            gci,
-		`^gci/([\w-]+)/(.+)$`:       gciCi,
-		`^ci/(.+)$`:                 ci,
-		`^release/(latest.*)$`:      rc,
-		`^release/(stable.*)$`:      stable,
-		`^(v\d+\.\d+\.\d+[\w.-]*)$`: version,
-		`^(gs://.*)$`:               gcs,
-		`^(bazel/.*)$`:              bazel,
+		`^(local)`:                     local,
+		`^gke-?(staging|test)?$`:       gke,
+		`^gci/([\w-]+)$`:               gci,
+		`^gci/([\w-]+)/(.+)$`:          gciCi,
+		`^ci/(.+)$`:                    ci,
+		`^release/(latest.*)$`:         rc,
+		`^release/(stable.*)$`:         stable,
+		`^(v\d+\.\d+\.\d+[\w\.\-+]*)$`: version,
```

Comment:
do you need the `\` before the `.`?

Reply:
isn't `.` by itself will match any single character?

Reply:
I think because this is a character class `.` doesn't have special meaning here - it's just a period, nothing else.

Reply:
Jeff can you suggest a unit test that demonstrates the problem?

Reply:
I did some one-off tests and I don't think having the `\` before the `.` has any effect, either negative or positive. I think @krzyzacy already added a unit test case which covers this.

Reply:
oh whoops, I missed the important `the`, yeah that probably don't take any effect, but `-` does